packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -144,20 +142,68 @@ router.put(
           cxId,
           id: facility.id,
           oid: facility.oid,
-          data: { name: cwOboOrgName, type: cxOrg.data.type, location: facility.data.address },
+          data: {
+            name: cwOboOrgName,
+            type: cxOrg.data.type,
+            location: addressStrict,
+          },
           organizationNumber: facility.facilityNumber,
-          eTag: "",
           createdAt: new Date(),
           updatedAt: new Date(),
         },
         true
       );
     }
 
-    return res.sendStatus(httpStatus.OK);
+    return res.status(httpStatus.OK).json(facility.dataValues);
   })
 );
 
+function getAddressFromInput(input: FacilityOboDetails): AddressStrict {
+  return {
+    addressLine1: input.addressLine1,
+    addressLine2: input.addressLine2,
+    city: input.city,
+    state: input.state,
+    zip: input.zip,
+    country: input.country,
+  };
+}
+
+function removeCoordinates(address: AddressWithCoordinates): AddressStrict {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { lat, lon, ...rest } = address;
+  return rest;
+}
+
+async function getAddress(
+  inputAddress: AddressStrict,
+  cxId: string
+): Promise<AddressWithCoordinates> {
+  const addresses = await addCoordinatesToAddresses({
+    addresses: [inputAddress],
+    cxId,
+  });
+  const address = (addresses ?? [])[0];
+  if (!address) throw new Error("Failed to geocode the address");
+  if (!address.coordinates) {
+    throw new MetriportError(`Missing coordinates for address`, undefined, {
```

Comment:
hypernit: would expect strings with no interpolation to be consistent on backticks vs quotes

Reply:
What's the difference? 😃 It's easier to type, it achieves the same result, it doesn't get converted by auto-formatting, one less thing to think about (consistency), and less adjusting if/when need to interpolate (move from double to backtick).

_(We talked about this before and ended up agreeing this was fine, btw 😄)_